pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -305,6 +304,32 @@ func TestRhpamProdwSmartRouter(t *testing.T) {
 	assert.Equal(t, fmt.Sprintf("rhpam%s-businesscentral-monitoring-openshift", cr.Spec.CommonConfig.Version), env.Console.DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Image)
 }
 
+func TestRhpamProdwSmartRouterWithSSL(t *testing.T) {
```

Comment:
is there a typo there?
TestRhpamProd**w**SmartRouterWithSSL

Reply:
it seems to, a extra w character.